SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON

```diff
@@ -53,6 +53,7 @@ systemtap
 SPEC files that are derived from OpenSuse SPEC files, which are under the license of the package itself, or MIT license if it the package is not under an open source license (https://en.opensuse.org/openSUSE:Specfile_guidelines):
 python-jinja2
 acl
+gd
```

Comment:
It looks like `gd` was taken from Photon. Why are we adding a line here?

Reply:
OK, now I'm even more confued. Joe's original changelog entry didn't say that `gd.spec` was taken from Photon. How did that got added now? And now I've also taken a look at the other entries and I see why you're saying it might have been taken from openSUSE. Please double-check, that this spec matches the one from OpenSUSE in that case and assign it accordingly.